### PR TITLE
Itr-4: feat: open modal window on click

### PR DESCRIPTION
**Pull request by MilaKolasava**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -4,11 +4,18 @@ import { ReactComponent as Loupe } from "../../assets/loupe.svg";
 import MenuItems from "../MenuItems/MenuItems";
 import { MENU_ITEMS } from "./Header.constant";
 
-function Header() {
+function Header(props: { setModalActive: (arg0: boolean) => void }) {
```
Comment (glebpigulevsky) [inline src/components/Header/Header.tsx]: please write interface for this instead of the cheating

File: src/App.tsx [hunk 2]
```diff
@@ -1,14 +1,15 @@
-import React from "react";
+import React, { useState } from "react";
 import Header from "./components/Header/Header";
 import Jumbotron from "./components/Jumbotron/Jumbotron";
 import ModalWindow from "./components/ModalWindow/ModalWindow";
 
 function App() {
+  const [modalActive, setModalActive] = useState(() => false);
```
Comment (glebpigulevsky) [inline src/App.tsx]: empty line before return

Comment (MilaKolasava) [reply src/App.tsx]: done

File: src/components/Header/Header.tsx (hunk 1 above)
Comment (MilaKolasava) [reply src/components/Header/Header.tsx]: done

**Review by glebpigulevsky — APPROVED** [review]

**Comment by glebpigulevsky** [comment]
good job!

**Pull request merged**